circle.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
     - pip install scipy
     - pip install scikit-learn
     - pip install nose-timer
+    - pip install memory_profiler psutil
```

Comment:
Maybe we don't really care about running memory_profiler tests on CircleCI, given that we are already running them on Travis.